shortfin/python/shortfin_apps/sd/server.py

```diff
@@ -403,6 +413,9 @@ def main(argv, log_config=UVICORN_LOG_CONFIG):
         help="Use tunings for attention and matmul ops. 0 to disable.",
     )
     args = parser.parse_args(argv)
+    if not is_port_valid(args.port):
+        exit(3)
```

Comment:
Getting flashbacks of "error 37" from a couple years ago but this is good enough for now.